cms/static/js/factories/settings.js

```diff
@@ -16,6 +16,7 @@ define([
         model = new CourseDetailsModel();
         model.urlRoot = detailsUrl;
         model.showCertificateAvailableDate = showCertificateAvailableDate;
+        model.set('upgrade_deadline', upgradeDeadline);
```

Comment:
What's `model.set` vs `model.upgrade_deadline =`? Also, should the first part be `upgradeDeadline` to stick with camel?

Reply:
So, this is some arcane backbone thing I don't fully understand. But just using `model.upgrade_deadline` doesn't properly put it into the model, where you can retrieve it with `get()`. The other properties above are merely being set for custom code later, not trying to be put into the model.

And I avoided camel case because of the rest of the data inside the model is snake case too.